frameworks/Python/uvicorn/uvicorn_conf.py

```diff
@@ -0,0 +1,13 @@
+import multiprocessing
+import os
+
+_is_travis = os.environ.get('TRAVIS') == 'true'
+
+workers = multiprocessing.cpu_count()
+if _is_travis:
+    workers = 2
+
+bind = "0.0.0.0:8080"
+keepalive = 120
+errorlog = '-'
+pidfile = 'uvicorn.pid'
```

Comment:
@nbrady-techempower Updated this PR for now. I'll aim to address the other Python framework pid files in due course.